lib/onelogin/ruby-saml/attributes.rb

```diff
@@ -1,91 +1,110 @@
 module OneLogin
   module RubySaml
-    # Wraps all attributes and provides means to query them for single or multiple values.
-    # 
-    # For backwards compatibility Attributes#[] returns *first* value for the attribute.
-    # Turn off compatibility to make it return all values as an array:
-    #    Attributes.single_value_compatibility = false
+
+    # SAML2 Attributes. Parse the Attributes from the AttributeStatement of the SAML Response. 
+    #
     class Attributes
       include Enumerable
 
       # By default Attributes#[] is backwards compatible and
       # returns only the first value for the attribute
-      # Setting this to `false` returns all values for an attribute
+      # Setting this to `false` returns all values for an attribute      
```

Comment:
nitpick: let's be careful with adding trailing whitespaces, might want to add a hook to your editor to remove these